src/Runner.Worker/Handlers/CompositeActionHandler.cs

```diff
@@ -274,9 +244,7 @@ private async Task RunStepAsync(IStep step)
             }
 
             Trace.Info($"Step result: {step.ExecutionContext.Result}");
-
-            // Complete the step context.
```

Comment:
This comment was copied from stepsrunner but doesn't make sense here. We never call `.Complete()` on the nested steps.